src/filtrering/filtrering-mine-filter/mine-filter-rad.tsx

```diff
@@ -1,19 +1,19 @@
-import {MineFilter} from '../../ducks/mine-filter';
 import {useDispatch, useSelector} from 'react-redux';
 import {AppState} from '../../reducer';
 import {finnSideNavn, mapVeilederIdentTilNonsens} from '../../middleware/metrics-middleware';
 import {logEvent} from '../../utils/frontend-logger';
 import {velgLagretFilter} from '../../ducks/filtrering';
-import {apneMineFilterModal} from '../../ducks/mine-filter-ui';
+import {apneMineFilterModal, markerValgtFilter} from '../../ducks/mine-filter-ui';
 import {Radio} from 'nav-frontend-skjema';
 import RedigerKnapp from '../../components/knapper/rediger-knapp';
 import React, {RefObject, useRef} from 'react';
 import {antallFilter} from '../../components/modal/mine-filter/mine-filter-utils';
 import {ListevisningType} from '../../ducks/ui/listevisning';
+import {LagretFilter} from "../../ducks/lagretFilter";
 
 interface LagretFilterRadProps {
-    filter: MineFilter;
-    filtergruppe: string;
+    filter: LagretFilter;
+    filtergruppe: ListevisningType;
```

Comment:
change filtergruppe to lagretFiltergruppe

Reply:
ee, i don't know about this one, since filtergruppe is more like page (min oversikt, enhetens...). What do you think?

Reply:
shit, that´s true! nevermind :D

Reply:
:)